tests/acceptance/features/apiEmailNotification/emailNotification.feature

```diff
@@ -0,0 +1,42 @@
+@api @email
+Feature: Email notification
+  As a user
+  I want to get email notification of actions related to me
+  So that I can stay updated about the information
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario: user gets email notification of space shares
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+    When user "Alice" shares a space "new-space" with settings:
+      | shareWith | Brian  |
+      | role      | Editor |
+    Then the HTTP status code should be "200"
+    And the email address of user "Brian" should have received an email for space "new-space" from user "Alice" with the body containing
```

Comment:
```suggestion
    And the user "Brian" should have received an email from user "Alice" for sharing space project "new-space" and body of email containing
```

may be something like this.

Reply:
```suggestion
    And the user "Brian" should have received an email from user "Alice" for sharing space project "new-space" 
```
maybe we could skip `and body of email containing`